docs/network/wwivnet_linux.md

```diff
@@ -103,105 +51,112 @@ dosemu -dumb -quiet "$(basename ${0}).bat ${NET}" 2>/dev/null
-the BBS, it has someplace to go (i.e., run DOS binaries).
+This allows calls from the BBS to the WWIVnet binaries to have somewhere to go.
 
-_**Configuring WWIVnet details and directories**_
+## Configuring WWIVnet
 
-1. create ${WWIV_DIR}/nets/wwivnet
-2. get network info files wwivnet.zip and extract into ${WWIV_DIR}/nets/wwivnet
-3. create ${WWIV_DIR}/nets/wwivnet/callout.net: 
- * Make a file with a single line containing the @1 node (it's the only active node relaying messages, atm).  The format is:
- * @1 & "yourpassword"
- * "yourpassword" is the one you provided to 1@1 and the file must include the double-quotes.
-4. use wwivconfig to set up network info (select "N")
- * Network type : WWIVnet
- * Network name : WWIVnet
- * Node number : -your node number-
- * Data Directory : nets/wwivnet/
-5. run network3 (the shell script) - Now that all the WWIVnet config bits are in place, we can run the network3 program to validate our setup.  If everything goes as expected, you should get a network report mailed to you on your board locally.  The command to run is:
- * network3 y .-your network position in wwivconfig-  
- for example, your list of nets in wwivconfig starts with 0, so your first network is .0, the second is .1, etc.  Since this is probably your first network, the command is probably network3 y .0
- * If no network number is given, .0 is assumed, so network3 y is the same as network3 y .0
+* Create a directory for WWIVnet: `mkdir -p ${WWIV_DIR}\nets\wwivnet` .
+* Get the latest version of [WWIVNET-###.zip](https://build.wwivbbs.org/jenkins/job/wwivnet/lastStableBuild/label=windows/) on the build server.  Do not worry that the build tag says "windows"; it's the same information.  
+* Put all of the contents of the zip in `${WWIV_DIR}/nets/wwivnet`
+* run ```wwivconfig``` and select N. Network Configuration
 
+```
+┌───────────────────────────────────────── Network Configuration ┐
+│ Net Type  : WWIVnet                                            │
+│ Net Name  : WWIVnet                                            │
+│ Node #    : YOURNODE                                           │
+│ Directory : nets\wwivnet\                                      │
+└────────────────────────────────────────────────────────────────┘
+```
 
-**Basic Mail Processing Workflow**  
-WWIVnet uses BinkP to transfer messages between systems.  In most cases, all you need to do is schedule the network binary to run periodically to push/pull messages.  It will call networkb as a sub-process to handle the BinkP connection.  If it finds any message files on your system to send out, it will pick them up and send to the target node.  After it is done sending, it will then grab anything on the target system that is destined for your board and insert them into the local mesg files. 
 
-The basic command to invoke the message transport is:
+* Create `callout.net` with `touch ${WWIV_DIR}/nets/wwivnet/callout.net`
```

Comment:
actually you don't need this anymore either, wwivconfig should do this for you and  you can edit the callout.net in the network tab of wwivconfig  -_ WDYT? Should we encourage that route instead?

Reply:
Edited; yes, that's probably best for the long run.